### PR TITLE
Adding documentation for Printing Focused option.

### PR DESCRIPTION
**Pull request by Matafou**
This was forgotten in the PR implementing this option.

### CODE REVIEW

File: doc/refman/RefMan-oth.tex [hunk 1]
```diff
@@ -974,7 +974,20 @@ \section{Controlling display}
   Printing Width} above).
 
 \subsection[\tt Test Printing Compact Contexts.]{\tt Test Printing Compact Contexts.\optindex{Printing Compact Contexts}}
-This command displays the current state of compaction of goal d'isolat.
+This command displays the current state of compaction of goal.
+
+
+\subsection[\tt Unset Printing Compact Contexts.]{\tt Unset Printing Unfocused.\optindex{Printing Unfocused}}
```
Comment (Zimmi48) [inline doc/refman/RefMan-oth.tex]: copy-paste error

File: doc/refman/RefMan-oth.tex [hunk 2]
```diff
@@ -974,7 +974,20 @@ \section{Controlling display}
...
+This command resets the displaying of goals to focused goals only
+(default at the time of writing this documentation). Unfocused goals
```
Comment (Zimmi48) [inline doc/refman/RefMan-oth.tex]: just say "default setting", if it changes, the documentation *must* be updated

File: doc/refman/RefMan-oth.tex [hunk 3]
```diff
@@ -974,7 +974,20 @@ \section{Controlling display}
...
+are create by focusing other goals with bullets or curly braces.
```
Comment (Zimmi48) [inline doc/refman/RefMan-oth.tex]: are created

File: doc/refman/RefMan-oth.tex [hunk 4]
```diff
@@ -974,7 +974,20 @@ \section{Controlling display}
...
+
+\subsection[\tt Set Printing Compact Contexts.]{\tt Set Printing Unfocused.\optindex{Printing Unfocused}}
```
Comment (Zimmi48) [inline doc/refman/RefMan-oth.tex]: same copy-paste error

File: doc/refman/RefMan-oth.tex [hunk 5]
```diff
@@ -974,7 +974,20 @@ \section{Controlling display}
...
+This command enables the displaying of unfocused goals. The goals are
+displayed after the focused ones and are distinguished by a separator.
+
+\subsection[\tt Test Printing Compact Contexts.]{\tt Test Printing Unfocused.\optindex{Printing Unfocused}}
```
Comment (Zimmi48) [inline doc/refman/RefMan-oth.tex]: same copy-paste error

**Comment by Matafou** [comment]
I fixed this problems, and added a reference to bullet documentation.

**Comment by Matafou** [comment]
The tests fail on coqdoc, is it a known issue or is it this PR that breaks. That would be surprising.

**Comment by Zimmi48** [comment]
Known issue: Due to frequent failures, we are currently disregarding the AppVeyor results and only focusing on Travis. Sorry about that. (It should get fixed, but after the beta.)

**Pull request merged**